src/components/Wallets/Wallet.vue

```diff
@@ -397,9 +412,22 @@ export default {
     },
 
     resetTransferForm () {
-      this.transferForm.to = ''
-      this.transferForm.amount = '0'
+      this.transferForm.to = null
+      this.transferForm.amount = null
       this.transferForm.description = ''
+    },
+
+    resetWithdrawForm () {
+      this.withdrawForm.amount = null
+      this.withdrawForm.wallet = null
+    },
+
+    validateForm (ref) {
+      let isValid = true
+      this.$refs[ref].validate(valid => {
+        if (!valid) isValid = false
+      })
+      return isValid
```

Comment:
Is validation already finished at this point? I mean, I guess `validate()` is asynchronous so `isValid` will never be false. 🤔

Reply:
Maybe I don't understand, but if I try to submit form that have not valid fields it will return false.


![image](https://user-images.githubusercontent.com/16295803/43772735-c61d4a20-9a4b-11e8-85c1-e9cca9226b1e.png)
On screenshot there is `console.log` for the returning result of `validateForm()`. If I don't fill any fields it will return false.

Reply:
Ok! So, which means that async-validator finishes validation immediately and a callback gets called synchronously. I guess that there is no problem as long as all validations are synchronous.